### DISCUSSION TITLE
Split View in context menu

### DISCUSSION THREAD

**Discussion by R0DR160HM** [Floorp-Projects/Floorp#1397] [Feature Request]
Hello, I've been using Floorp for a few months and loving it so far, you guys are doing a great job.

After the recent 11.17 update, the Split View functionality became very useful and I think it would be even better (and far more ergonomic) if you could just right click on the page (not on the tab, but on the page itself) and have the option to Open in Split View. That would be particularly useful for users that have the tabs on a sidebar.

I know this is nitpicking and wouldn't necessarily change the experience of most users at all but it would speed up the browsing experience for power users.

**Comment by Ryosuke-Asano** [comment]
I'll consider it.
